app/views/practices/_search.js.erb

```diff
@@ -86,67 +86,75 @@ function buildPracticeCard(result) {
 
+    var cardRetiredBanner = '';
     if (result.item.retired) {
-        noCardImageHtml += retiredBanner;
+        cardRetiredBanner = '<div class="dm-practice-retired-banner"><h5>Retired</h5></div>';
     }
-
-    noCardImageHtml += '</div>';
-    var cardImage = practiceImage ? cardImageHtml : noCardImageHtml;
-    var assignPracticeHeader = practiceImage ? cardHeaderHtml : '';
+    
 
     cardHtml +=
         '<div class="dm-practice-card padding-bottom-3 tablet:grid-col-6 desktop:grid-col-4 position-relative">' +
-        '<a href="/innovations/' + result.item.slug + '" tabindex="-1" aria-hidden="true" class="dm-practice-link-aria-hidden position-absolute top-0 bottom-3 left-105 right-105 z-100"></a>' +
-        '<div class=" dm-practice-card-container">' +
-        cardImage +
-        '<div class="padding-105 height-card-mobile">' +
-        assignPracticeHeader +
-        '<div>' +
-        '<p class="multiline-ellipses-1 font-sans-sm practice-card-tagline">' + result.item.tagline + '</p>' +
-        '</div>' +
-        '<div class="dm-practice-card-origin margin-bottom-2">' +
-        '<span class="font-sans-2xs text-italic line-height-sans-3 dm-practice-card-origin-info">' +
-        'Created in ' + result.item.date_initiated + ' ' + result.item.initiating_facility_name +
-        '</span>' +
-        '</div>' +
-        '</div>' +
-        '</div>' +
+            '<a href="/innovations/' + result.item.slug + 
+                '" tabindex="-1" aria-hidden="true" class="dm-practice-link-aria-hidden position-absolute top-0 bottom-3 left-105 right-105 z-100"></a>' +
+            '<div class="dm-practice-card-container">' +
+                cardImagePlaceholder +
+                cardRetiredBanner +
+                '<div class="padding-105 height-card-mobile">' +
+                    cardHeaderHtml +
+                    '<p class="multiline-ellipses-1 font-sans-sm practice-card-tagline">' + result.item.tagline + '</p>' +
+                    '<div class="dm-practice-card-origin margin-bottom-2">' +
+                        '<span class="font-sans-2xs text-italic line-height-sans-3 dm-practice-card-origin-info">' +
+                            'Created in ' + result.item.date_initiated + ' ' + result.item.initiating_facility_name +
+                        '</span>' +
+                    '</div>' +
+                '</div>' +
+            '</div>' +
         '</div>';
 
     return cardHtml;
 }
 
+function replacePlaceholderWithImage(imageUrl, practiceId, practiceName) {
+    loadImage(imageUrl, function(loadedImageSrc) {
+        var imgTag =
+            '<div class="dm-practice-card-img-container">' +
+                '<img data-resource-id="' +
+                practiceId +
+                '" src="' +
+                loadedImageSrc +
+                '" alt="' + practiceName + ' Marketplace Card Image" ' +
+                'width="247.5" height="165" ' +
```

Comment:
Sorry! I know I suggested this, but after reviewing in DEV, I'm realizing that explicitly setting the width in the HTML will make the images look stretched out in mobile widths (image on the left is DEV, image on the right is PROD). Let's hack on [DM-4125](https://agile6.atlassian.net/browse/DM-4125?atlOrigin=eyJpIjoiM2QyZTA2OGE5YmE0NDVlZGEzNjdhZmZmYjY1MTkzMGUiLCJwIjoiaiJ9) in a separate PR instead.
<img width="757" alt="responsive-pictures" src="https://github.com/department-of-veterans-affairs/diffusion-marketplace/assets/5402927/c1e48cda-04ce-407e-922e-91dc7ec25d47">